crates/protocol/src/info/ecotone.rs

```diff
@@ -64,6 +62,11 @@ impl L1BlockInfoEcotone {
     /// The 4 byte selector of "setL1BlockValuesEcotone()"
     pub const L1_INFO_TX_SELECTOR: [u8; 4] = [0x44, 0x0a, 0x5e, 0x20];
 
+    /// Returns whether the scalars are empty.
+    pub const fn empty_scalars(&self) -> bool {
+        self.base_fee_scalar == 0 && self.blob_base_fee_scalar == 0
```

Comment:
makes sense I guess 
```suggestion
        self.base_fee_scalar == 0 && self.blob_base_fee_scalar == 0 && self.l1_fee_overhead != 0
```
this would make it totally safe right? it means (`base_fee_scalar`, `blob_base_fee_scalar`) is mutually exclusive to `l1_fee_overhead`

Reply:
Technically I believe the l1 fee overhead _could_ be zero if configured as such

Reply:
ok, good catch!

Reply:
would be nice to eternalise that in a code comment, bet I won't be the first dev asking myself this